tests/errmsgs/t10734.nim

```diff
@@ -0,0 +1,18 @@
+discard """
+  cmd: "nim check $file"
+  errormsg: ""
+  nimout: '''t10734.nim(17, 1) Error: invalid indentation
```

Comment:
general comment: always prefer newline after triple double quote string and triple single quote string, it's the same (see spec) but is more readable:
```
  nimout: '''t10734.nim(17, 1) Error: invalid indentation
```
=>

```
  nimout: '''
t10734.nim(17, 1) Error: invalid indentation
```